### PR TITLE
Update xlplayer from 2.0.3.1265 to 2.0.5.1285

### PR DESCRIPTION
**Pull request by ran-dall**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by ran-dall** [comment]
@suschizu I don't know why Travis has trouble connecting.

**Comment by vitorgalvao** [comment]
> I don't know why Travis has trouble connecting.

I’m having trouble too.

**Comment by ran-dall** [comment]
@vitorgalvao I can't connect ethier so I'll just close this. @suschizu can open a new one if you want to update it.

File: Casks/xlplayer.rb [hunk 1]
```diff
@@ -1,13 +1,13 @@
 cask 'xlplayer' do
-  version '2.0.3.1265'
-  sha256 'fbdf478d9dcaa7aba0f1cf7c212993edd0af7ba9a520e4ae4dd84ccf843690c0'
+  version '2.0.5.1285'
+  sha256 '80136420cf7e38279af00009ca487b131ae82a7031964488d907287ae2b855a6'
 
   # down.sandai.net was verified as official when first introduced to the cask
   url "http://down.sandai.net/mac/xlplayer_#{version}.dmg"
-  appcast 'https://video.xunlei.com/mac.html'
+  appcast 'http://video.xunlei.com/mac.html'
```
Comment (vitorgalvao) [inline Casks/xlplayer.rb]: Figured out the issue.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**